src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Comment } from '../../typedefs/Comment';
+import '../CommentList/CommentList.scss';
+
+interface Props {
+  singleComment: Comment
+}
+
+export const CommentInfo: React.FC<Props> = ({ singleComment }) => {
+  const {
+    body,
+    name,
+    email,
+  } = singleComment;
```

Comment:
```suggestion
export const CommentInfo: React.FC<Props> = ({ body, name, email }) => {
...
)
```
I think it's better to pass each prop when we call the component, so we can destructure props fields on arguments

Reply:
This could be better if we would receive only one object as a parameter. But we receive the object as a parameter and restructure it already. This object has another object as a field. And we need properties from this one. How can we restructure right in parameters? In your suggestion, you're destructing the object, but it has no keys _body, _name_, and _email_.

Reply:
Yeah, so you need to pass each prop when you call CommentInfo from CommentList component. 
Check this [video](https://www.youtube.com/watch?v=LGqqEg1Z4FA), timecode - 2:58